rpcs3/rpcs3qt/log_frame.cpp

```diff
@@ -537,6 +606,15 @@ void log_frame::UpdateUI()
 		if (steady_clock::now() >= start + 4ms || buf.empty()) break;
 	}
 
+	if (!m_log && !m_plain_log)
+	{
+		return;
+	}
+
+	const auto font_start_tag = [](const QColor& color) -> const QString { return "<font color = \"" + color.name() + "\">"; };
+	const QString font_start_tag_stack = "<font color = \"" + m_color_stack.name() + "\">";
+	const QString font_end_tag = "</font>";
```

Comment:
```suggestion
	const QString font_end_tag = QStringLiteral("</font>");
```

If only multiline suggestions were supported...